app/api/resources/upload/route.ts

```
@@ -31,9 +31,24 @@ export const config = {
 export async function POST(request: NextRequest) {
   const { userId } = await auth();
-
-  if (!userId) {
-    return NextResponse.json({ error: "Unauthorized" }, { status: 401 });
+  const searchParams = request.nextUrl.searchParams;
+  const sessionId = searchParams.get("sessionId");
+
+  // Log auth state for debugging
+  console.log("POST /api/resources/upload auth:", {
+    userId,
+    sessionId: sessionId ? sessionId.substring(0, 8) + "..." : null,
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)